core/src/components/slides/slides.tsx

```diff
@@ -171,6 +174,10 @@ export class Slides implements ComponentInterface {
 
   /**
    * Transition to the specified slide.
+   *
+   * @param index The index of the slide to transition to.
+   * @param speed The transition duration (in ms).
+   * @param runCallbacks If true, the transition will produce transition events.
```

Comment:
What is a "transition event"? Is that just an event that's fired when the transition finishes?